pom.xml

```diff
@@ -303,30 +430,20 @@
                 </executions>
             </plugin>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
+                <!-- This replaces unsupported findbugs plugin -->
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <version>4.2.0</version>
                 <executions>
                     <execution>
                         <id>finding-bugs</id>
                         <phase>compile</phase>
                         <goals>
                             <goal>check</goal>
                         </goals>
-                        <configuration>
-                            <threshold>Low</threshold>
-                            <xmlOutput>true</xmlOutput>
-                            <failOnError>true</failOnError>
-                        </configuration>
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
What is the reason for removing release plugin?

Reply:
Release plugin is already configured in jboss-parent pom. There is IMHO no reason why to configure it here since AFAIK the testsuite had never seen a release.

Reply:
> Release plugin is already configured in jboss-parent pom.

Well, it seems that it is defined in plugin management section, not a plugin section.

> There is IMHO no reason why to configure it here since AFAIK the testsuite had never seen a release.

Agree, resolving this thread